tools/cli/wsk

```diff
@@ -154,6 +154,8 @@ def propCmd(args, props, userprops, propsLocation):
             wskprop.updateProps('APIVERSION', args.apiversion, propsLocation)
             print 'ok: whisk API version set'
         if args.namespace:
+            if args.apihost is not None:
```

Comment:
Not necessary. If the user tries to set their API host and namespace at the same time they'll get error (usually that their authentication is not valid). This fix corrects it but this is not common IMO.